src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Windows/SplashScreen.cs

```diff
@@ -285,144 +293,145 @@ private void Fadeout_Tick(object unused, EventArgs args)
             }
         }
 
-        private void DestroyResources()
+        ~SplashScreen() => DestroyResources(finalizer: true);
+
+        private unsafe void DestroyResources(bool finalizer = false)
```

Comment:
📝 this parameter appears to be inverted relative to standard practices.

Reply:
The standard practice is to have a dispose. This shipped so that would require an API change. I deliberately was trying to demonstrate how to use CsWin32 the way WinForms does so that we can align for better productivity. I tried to avoid other refactoring to make the code cleaner.